lib/Agrammon/DB/Dataset.pm6

```diff
@@ -217,16 +217,14 @@ class Agrammon::DB::Dataset does Agrammon::DB {
         }
     }
 
-    method store-input-comment($var-name, $comment) {
+    method store-input-comment(:$variable, :$comment) {
```

Comment:
Named parameters are optional by default; is that what you wanted?

Reply:
Oh, `$variable` should not be optional. `$comment` should be optional to be able to "clear" the comment.

Reply:
Fixed.